pytorch/training/docker/1.8.0/py3/Dockerfile.cpu

```diff
@@ -0,0 +1,132 @@
+FROM ubuntu:18.04
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+# Add arguments to achieve the version, python and url
+ARG PYTHON_VERSION=3.6.13
+ARG OPEN_MPI_VERSION=4.0.1
+
+# The smdebug pipeline relies for following format to perform string replace and trigger DLC pipeline for validating
+# the nightly builds. Therefore, while updating the smdebug version, please ensure that the format is not disturbed.
+ARG SMDEBUG_VERSION=1.0.1
```

Comment:
Can we upgrade `smdebug` to `v1.0.5`?

Reply:
It is safer to keep v1.0.1 for now.
We have noticed a mismatching issue before between smdebug & horovod. 
We could update this the future.